### PR TITLE
#8584 Optimize code

### PR DESCRIPTION
**Pull request by KenHoang16CDTH12**
#Bug 8584
https://edu-redmine.sun-asterisk.vn/issues/8584

- Tối ưu code.

### CODE REVIEW

**Comment by KenHoang16CDTH12** [comment]
ready

File: app/src/main/java/com/sun/music61/util/notification/MusicNotificationHelper.java [hunk 1]
```diff
@@ -108,11 +111,12 @@ private void loadImageSong(Track track) {
                     @Override
                     public void onCompleted(Bitmap bitmap) {
                         mBuilder.setLargeIcon(bitmap);
-                        MusicNotificationHelper.this.notify(mBuilder.build());
+                        handleStartAndStopForeground();
                     }
 
                     @Override
                     public void onFailure() {
+                        handleStartAndStopForeground();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/music61/util/notification/MusicNotificationHelper.java]: 114, 119 lặp code

File: app/src/main/java/com/sun/music61/screen/service/PlayTrackService.java [hunk 2]
```diff
@@ -200,7 +214,7 @@ public int getShuffle() {
 
     public void loopTracks(@Loop int loop) {
         mPlayerManager.setLoop(loop);
-        notifySettingChange();
+        update(!Status.HAVE_UPDATE_STATE, Status.HAVE_UPDATE_SETTING, !Status.HAVE_UPDATE_TRACK, !Status.HAVE_UPDATE_NOTIFICATION);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/music61/screen/service/PlayTrackService.java]: lặp code
refactor code đi e

**Comment by KenHoang16CDTH12** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**